### PR TITLE
[Merged by Bors] - Fix failing TestBuilder_PublishActivationTx_HappyFlow

### PR DESCRIPTION
**Pull request by nkryuchkov**
## Motivation

WIP on https://github.com/spacemeshos/pm/issues/40

## Changes

Fix failing `TestBuilder_PublishActivationTx_HappyFlow`

## Test Plan

- [x] `TestBuilder_PublishActivationTx_HappyFlow` passes

## TODO

- [x] Finish

## DevOps Notes

- [ ] Does this code require configuration changes?
- [ ] Does this code affect public APIs?
- [ ] Does this code assume a new version of external services (PoET, elasticsearch, etc.)
- [ ] Does this code make changes to log messages that our monitoring infrastructure may rely on?

### CODE REVIEW

**Comment by nkryuchkov** [comment]
bors try

File: activation/activation_test.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func (l *LayerClockMock) GetCurrentLayer() types.LayerID {
 func (l *LayerClockMock) AwaitLayer(types.LayerID) chan struct{} {
 	ch := make(chan struct{})
 	go func() {
-		time.Sleep(1 * time.Millisecond)
+		time.Sleep(1 * time.Second)
```
Comment (noamnelke) [inline activation/activation_test.go]: For something that's flaky (not failing consistently), is 1000x increase really required? Maybe 50ms will be enough to make it stable?

Comment (nkryuchkov) [reply activation/activation_test.go]: @noamnelke After running a few million iterations, I didn't see anything higher than 150ms. So, I rounded it up to the next power of 10. With lower values, the test can fail rarely, but still can.

Comment (nkryuchkov) [reply activation/activation_test.go]: @noamnelke Do you think something like 200ms or 500ms is worth trying? Or doesn't it make much difference if we increase the timeout so much?

**Review by antonlerner — APPROVED** [review]

File: activation/activation_test.go (hunk 1 above)
Comment (noamnelke) [reply activation/activation_test.go]: This is not a timeout, but an actual sleep. A second isn't that much, so I leave it up to you, but based on what you say, maybe 300-500ms?

**Review by noamnelke — APPROVED** [review]

File: activation/activation_test.go (hunk 1 above)
Comment (nkryuchkov) [reply activation/activation_test.go]: Yes, something times out because of it, so I meant this sleep. 500ms seems to work fine for 10K iterations

**Comment by nkryuchkov** [comment]
bors try

**Comment by nkryuchkov** [comment]
bors merge

**Pull request closed without merging**